man/avocado.rst

```diff
@@ -6,8 +6,8 @@
 SYNOPSIS
 ========
 
- avocado [-h] [-v] [--config CONFIG_FILE]
- {run,list,sysinfo,multiplex,plugins,datadir} ...
+ avocado [-h] [-v] [--config [CONFIG_FILE]] [--show [STREAM[:LVL]]] [-s]
```

Comment:
there is (and was) extra space at the beginning which increases the offset of the first paragraph.